website/cache/cve_cache.py

```
@@ -0,0 +1,139 @@
+        cvss_metric_v = next(iter(metrics))
+        cvss_data = metrics[cvss_metric_v]
+        
+        if not cvss_data or len(cvss_data) == 0:
+            return None
+        
+        base_score = cvss_data[0].get("cvssData", {}).get("baseScore")
+        
+        if base_score is not None:
+            return Decimal(str(base_score))
+        
+        return None
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)